### PR TITLE
[Mux/Merge] remove unnecessary value

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. remove unnecessary value in pad-data structure
2. add cond to check valid buffer

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
Please hold and do not merge until we start 0.0.3-1rc1

File: gst/tensor_mux/gsttensormux.c [hunk 1]
```diff
@@ -104,6 +102,7 @@ static void gst_tensor_mux_set_property (GObject * object, guint prop_id,
     const GValue * value, GParamSpec * pspec);
 static void gst_tensor_mux_get_property (GObject * object, guint prop_id,
     GValue * value, GParamSpec * pspec);
+static void gst_tensor_mux_finalize (GObject * object);
```
Comment (myungjoo) [inline gst/tensor_mux/gsttensormux.c]: Is ther any reason to move the declaration?

File: gst/tensor_merge/gsttensormerge.c [hunk 2]
```diff
@@ -106,6 +104,7 @@ static void gst_tensor_merge_set_property (GObject * object, guint prop_id,
     const GValue * value, GParamSpec * pspec);
 static void gst_tensor_merge_get_property (GObject * object, guint prop_id,
     GValue * value, GParamSpec * pspec);
+static void gst_tensor_merge_finalize (GObject * object);
```
Comment (myungjoo) [inline gst/tensor_merge/gsttensormerge.c]: Is there any reason to move this line?

**Comment by myungjoo** [comment]
I will merge this after #649

**Comment by leemgs** [comment]
> I will merge this after #649

#649 is merged. Let's merge this PR.

**Pull request merged**